### PR TITLE
Fix focus tracking for dynamic iframe content

### LINKED ISSUE #5882: useFocusRing not working inside iframe
### Provide a general summary of the issue here

In the latest release 3.32.0, react-aria added support for iframe tracking but it is not working correctly.

### 🤔 Expected Behavior?

useFocusRing should work inside a dynamically created iframe.

### 😯 Current Behavior

useFocusRing doesn't works. It never assumes that the element is focused.

### 🖥️ Steps to Reproduce

Here is a reproduction

https://codesandbox.io/p/sandbox/keen-julien-55yhq6?file=%2Fsrc%2FApp.js%3A33%2C9

### Version

3.32.0

### What browsers are you seeing the problem on?

Chrome

### What operating system are you using?

MacOS

### PR DESCRIPTION
**Pull request by ritz078**
Closes #5882 

## ✅ Pull Request Checklist:

- [x] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/WAI/ARIA/apg/)

### CODE REVIEW

**Comment by snowystinger** [comment]
GET_BUILD

**Review by snowystinger — APPROVED** [review]
looks good, if you have some time, a unit test would be good too, to make sure we don't have regressions. the story is helpful, thanks for including that

reminder for anyone looking at this. first click into an iframe will result in focus visible, this is expected. click a second time and it'll go away

File: packages/@react-aria/interactions/test/useFocusVisible.test.js [hunk 1]
```diff
@@ -284,6 +295,23 @@ describe('useFocusVisible', function () {
...
+      });
+      addWindowFocusTracking(iframeRoot);
+
+      const el = document.querySelector('iframe').contentWindow.document.body.querySelector('button[id="iframe-example"]');
+
+      fireEvent.focus(iframe.contentWindow.document.body);
+      fireEvent.keyDown(el, {key: 'Tab'});
+
+      expect(el.textContent).toBe('example-focusVisible');
+    });
```
Comment (ritz078) [inline packages/@react-aria/interactions/test/useFocusVisible.test.js:314]: @snowystinger I am not able to make this test work. Any idea what I might be doing wrong?

**Review by snowystinger — COMMENTED** [review]
Thanks for getting the test started. It appears there was an issue with jsdom tab/focus. I found a way to work around it.

**Review by snowystinger — APPROVED** [review]

**Review by yihuiliao — APPROVED** [review]

**Pull request merged**